packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/collapsible-content.twig

```diff
@@ -11,6 +11,7 @@
     <button class='ma__collapsible-content__toggle-all ma__collapsible-content__toggle-all--collapsed'>
```

Comment:
FYI, just noticed this button needs `aria-expanded` to tell screen readers its state just like buttons for individual accordions.